### PR TITLE
Adds test coverage to Cython files

### PR DESCRIPTION
**Pull request by Fenil3510**
Changes made in this Pull Request:
 - Added test coverage for cython, works fine locally. Minor changes from PR #2029. This looks kind of a hacky way to me, please review

PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

**Comment by richardjgowers** [comment]
@Fenil3510 we can't add the c/cpp files to ignore because we *do* include them in the master branch.  So releases don't require Cython.

File: package/setup.py [hunk 1]
```diff
@@ -94,6 +94,7 @@
               "parallelization module".format(
                Cython.__version__, required_version))
         cython_found = False
+    cython_linetrace = True
```
Comment (richardjgowers) [inline package/setup.py]: Ideally this isn't on by default as I'm guessing it has a performance cost.  We could probably look for one of the travis/CI environment variables to trigger this?

Comment (Fenil3510) [reply package/setup.py]: Yes, this is not a default one and there's an environment variable to trigger this as you had already done in #2029, but I was forcing this to look at the results for now.

**Comment by Fenil3510** [comment]
`.pyx` still not visible in `codecov` report. Have to work on it.

**Comment by Fenil3510** [comment]
> @Fenil3510 we can't add the c/cpp files to ignore because we _do_ include them in the master branch. So releases don't require Cython.

The `.c` files were due to local build, will have to remove them anyway.

**Pull request closed without merging**